packages/app/documents/service.ts

```diff
@@ -116,6 +116,35 @@ export async function createDocument(
         const [last] = insertedDocument['x-meditor'].states.slice(-1)
         const targetState = last.target
 
+        // Get the optional webhook URL from the workflow.
+        const [firstCurrentEdge] = modelWithWorkflow.workflow.currentEdges
+        const [_webhookConfigError, webhookConfig = null] = getWebhookConfig(
+            firstCurrentEdge.webhookURL
+        )
+
+        // If there is a webhook URL, get the underlying webhook config and invoke it.
+        if (webhookConfig) {
+            const [_error, response] = await invokeWebhook(webhookConfig, {
+                model: modelWithWorkflow,
+                document: insertedDocument,
+                state: targetState,
+            })
+
+            // NOTE: Right now, we're not doing anything with the webhook response; a future story would be to include this and the previous document status notifications into document history. While we're changing document history, we should make sure that "Init" state is represented:
+            /** 
+            "states": [
+                {
+                    "source": "Init",
+                    "target": "Draft",
+                    "modifiedOn": "2027-08-03T16:57:26.401Z",
+                    "modifiedBy": "username"
+                }
+            ]
+            */
+
+            log.debug(response)
```

Comment:
This was intentional, but it's not necessary.

Reply:
Can we actually change _error to error and log the error here if it exists? During testing, it wasn't immediately clear what was failing with my .env config